input/pagecontent/general_guidance.md

```diff
@@ -76,10 +76,16 @@ Advance directive documents may take several forms including scanned PDF documen
 
 The DocumentReference is the resource that is used for "indexing" of documents and can be used for searching and finding documents with specific attributes such as type of document, subject, or dates.
 
-<img src="./ADI_profile_resource_relationships.png" alt="Structure and Resource Relationships"  style="width: 100%; float: none; align: middle;"/>
+<img src="./ADI_profile_resource_relationships.png" alt="Structure and Resource Relationships"  style="width: 70%; float: none; vertical-align: middle; margin: auto"/>
 
 Digital signatures are defined as optional in this guide. If supported, the digital signature will be a captured in a `Binary` resource that is referenced by an additional `DocumentReference` resource.
 
+<blockquote class="stu-note">
+    <p>
+    There is movement within the FHIR community to move towards using the Provenance resource as a way to apply a digital signature to a document in FHIR.  The signature within the Provenance resource would be used to encrypt the binary (the actual document itself), not the DocumentReference.
+    </p>
+</blockquote>
+
 ### ADI FHIR Document Structure
 
 ADI FHIR native documents are instances of the `Bundle` resource with the `type` = `document`. The document should have all content contained within the Bundle with no external references except for the references to external documents in the [DocumentationObservation](StructureDefinition-ADI-DocumentationObservation.html).focus. FHIR `Bundle` documents consist of multiple entry resources within it, with the first entry being a `Composition` resource. The `Composition` resource acts as the header and organizational construct. It contains information about the document such as the category of document, dates, and references to the various participants of the document, as well as document sections used to categorize or organize the contained entries.
```

Comment:
small change but it would be more accurate to state [DocumentationObservation](StructureDefinition-ADI-DocumentationObservation.html) profile `Observation.focus` element.
Or alternately, remove the dot after DocumentationObservation and then state the `focus` element.